notices.json

```diff
@@ -1,38 +1,4 @@
 [
-  {
-    "id": "use_case_survey",
-    "conditions": {
-      "displayDate": "<= 2022-08-01T00:00:00Z"
-    },
-    "localizedMessages": {
-      "en": {
-        "title": "We want your feedback!",
-        "description": "We are looking for feedback on how we are doing in supporting your team's collaboration throughout different phases of the DevOps lifecycle. Please help us with a quick 5 minutes survey for a chance to win a set of AirPod Pros.",
-        "image": "https://raw.githubusercontent.com/mattermost/notices/master/images/Feedback.png",
-        "actionText": "Give feedback",
-        "actionParam": "https://forms.gle/otcG8ETaG8URZtEt8"
```

Comment:
I can see this form is still active. Should this be decommissioned or taken out of public reach if not to be used?